bin/pakiti-client

```diff
@@ -238,6 +238,26 @@ sub find_packages ($) {
     my($data) = @_;
     my($cmd, $format, @list);
     my(@output);
+	
+    #The user wants to send svmon data
+    $cmd = $Option{"svmonreport"};
+    if ($cmd) {
```

Comment:
If you change these two lines to:
  if ($Option{"svmonreport"}) {
we'll gain quite some readability (without misusing the $cmd variable ;-)